src/main/scala/mesosphere/marathon/api/v2/TasksResource.scala

```diff
@@ -91,14 +91,27 @@ class TasksResource @Inject() (
 
   @GET
   @Produces(Array(RestResource.TEXT_PLAIN_LOW))
-  def indexTxt(@Context req: HttpServletRequest, @Suspended asyncResponse: AsyncResponse): Unit = sendResponse(asyncResponse) {
+  def indexTxt(
+    @DefaultValue(MarathonCompatibility.Latest)@QueryParam("compatibilityMode") compatibilityMode: String = MarathonCompatibility.Latest,
```

Comment:
How should we check for `deprecatedFeatureSet.isEnabled(marathon14tasks)` here? Should we return an `HTTP 400`?